tests/containers/containers_3rd_party.pm

```diff
@@ -64,8 +64,10 @@ sub run {
     my ($running_version, $sp, $host_distri) = get_os_release;
 
     # Define test images here
-    my @docker_images = ('alpine', 'opensuse/leap', 'opensuse/tumbleweed', 'debian', 'ubuntu', 'centos', 'fedora');
-    my @podman_images = ('alpine', 'opensuse/leap', 'opensuse/tumbleweed', 'debian', 'ubuntu', 'centos', 'fedora');
+    my $ex_reg        = get_var('REGISTRY', 'docker.io') . '/library';
```

Comment:
Technically `library` is part of the image name, so I'd move the `library` part to the image list

Reply:
Yeah, it was just to avoid some text later. Maybe renaming the variable name?

Reply:
That would work too. `$os_prefix`/`$tp_prefix` maybe ("third-party")?